controllers/cloud.redhat.com/providers/autoscaler/wrapper.go

```diff
@@ -0,0 +1,21 @@
+package autoscaler
+
+import (
+	crd "github.com/RedHatInsights/clowder/apis/cloud.redhat.com/v1alpha1"
+	"github.com/RedHatInsights/clowder/controllers/cloud.redhat.com/config"
+	"github.com/RedHatInsights/clowder/controllers/cloud.redhat.com/providers"
+)
+
+type autoScaleProvider struct {
+	providers.Provider
+	Config config.DatabaseConfig
+}
+
+// NewNoneDBProvider returns a new none db provider object.
+func NewAutoScaleWrapper(p *providers.Provider) (providers.ClowderProvider, error) {
```

Comment:
I don't remember seeing this used anywhere - I think the file is perhaps old and mistakenly added here.

Reply:
Yup, you're right. Removed.